praw/models/reddit/subreddit.py

```diff
@@ -1393,6 +1405,19 @@ def update(
             "text_color": text_color,
             "text_editable": text_editable,
         }
+        if fetch:
+            _existing_data = [
+                template
+                for template in iter(self)
+                if template["id"] == template_id
+            ]
+            if len(_existing_data) == 0:
```

Comment:
Just as a sanity check, I'd prefer this to be `!= 1`.

Reply:
Sure.